src/test/java/io/antmedia/test/MuxerUnitTest.java

```diff
@@ -4555,4 +4555,11 @@ public void testSetSEIData() {
 		muxAdaptorReal.addSEIData(data);
 		verify(hlsMuxer, times(1)).setSeiData(data);
 	}
+
+	@Test
+	public void testRecordingWithRecordingSubfolder() {
+		appSettings.setRecordingSubfolder("records");
+		testMp4Muxing("record" + RandomUtils.nextInt(0, 10000));
```

Comment:
Please correct me if I'm wrong. We're not checking if it's recorded in the correct folder.